### PR TITLE
frontend: Updates text for experimental operator

### PR DESCRIPTION
**Pull request by amrutac**
Implements https://github.com/coreos/tectonic-installer/issues/794
screen shot 2017-05-22 at 4 48 14 pm

### CODE REVIEW

**Comment by robszumski** [comment]
What does Tectonic link to?

Is it possible to nest the placeholder text beneath the main text?

Can you click on the entire link of black text to check the box?

**Comment by amrutac** [comment]
@robszumski: It's the link to Tectonic Operators and the GA event call, it existed previously, so I left it in there. Let me know and I can get rid of it. The helper-text is aligned with other helper text on the page. I could add some padding there.

**Comment by robszumski** [comment]
> It's the link to Tectonic Operators and the GA event call, it existed previously, so I left it in there. Let me know and I can get rid of it.

yeah we can remove that

> The helper-text is aligned with other helper text on the page. I could add some padding there.

Yes please. Thanks!

**Comment by amrutac** [comment]
@robszumski 
screen shot 2017-05-22 at 7 24 34 pm

**Comment by robszumski** [comment]
One last change, I think the rest of the black text is 15px. This is my fault, I thought we used 16px. Can you change that, then LGTM?

**Comment by amrutac** [comment]
@robszumski: It's all set to 16px, I checked it yesterday
screen shot 2017-05-23 at 8 12 16 am

screen shot 2017-05-23 at 8 13 11 am

**Comment by amrutac** [comment]
ok to test

File: installer/frontend/components/experimental-features.jsx [hunk 1]
```diff
@@ -3,27 +3,21 @@ import { connect } from 'react-redux';
 import { CheckBox, WithClusterConfig } from './ui';
 
 import { UPDATER_ENABLED } from '../cluster-config';
-import { Alert } from './alert';
-import { TectonicGA } from '../tectonic-ga';
 
 export const ExperimentalFeatures = connect(
```
Comment (kans) [inline installer/frontend/components/experimental-features.jsx]: Since you no longer use props.updater, you should drop the call to connect.

Comment (amrutac) [inline installer/frontend/components/experimental-features.jsx]: Oh, you are right, I saw line 14 and thought it was still getting used.

**Comment by amrutac** [comment]
@kans: RFR

**Comment by amrutac** [comment]
ok to test

**Comment by sym3tri** [comment]
ok to test

**Comment by amrutac** [comment]
ok to test

**Pull request merged**